src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -440,7 +441,10 @@ public function thisOrderIsAlreadyPaid(OrderInterface $order)
 
     /**
      * @Given /^the customer canceled (this order)$/
```

Comment:
Since you're double elling stuff, might as well do it here and in the method's name.